codegen/src/main/twirl/templates/JavaClient/Client.scala.txt

```diff
@@ -110,7 +112,7 @@ public final class @{service.name}Client implements @{service.name}, AkkaGrpcCli
       public StreamResponseRequestBuilder<Source<@method.inputTypeUnboxed, NotUsed>, @method.outputTypeUnboxed> @{method.name}()
     }
     {
-      return @{method.name}RequestBuilder;
+      return clientState.withChannel( this::@{method.name}RequestBuilder);
```

Comment:
This doesn't work for Scala 2.11, because "scala.Function1 is not a functional interface" in Scala.11: https://travis-ci.com/playframework/play-grpc/jobs/171208483#L542

I'm not sure why this didn't fail in CI.

Reply:
It looks like, Scala 2.11 is not exercised on CI.

Reply:
Yep: https://github.com/akka/akka-grpc/issues/498 https://github.com/akka/akka-grpc/issues/497